README.md

```diff
@@ -866,6 +867,18 @@ _\* If no region is configured via the preset config, `hops-lambda` will try to
 
 For more details and more advanced use-cases, head over to the [full readme of `hops-lambda`](https://github.com/xing/hops/tree/master/packages/lambda).
 
+#### [`jest-preset-hops`](https://github.com/xing/hops/tree/master/packages/jest-preset)
+
+Hops uses wepback to configure and instrument your application. However, test runners aren't compatible with this setup out of the box.
+
+`jest-preset-hops` is a [Jest preset](https://jestjs.io/docs/en/configuration.html#preset-string) that allows to write tests for Hops applications. It aims to support all features that the official `hops-*` packages provide (e.g. transpiling your files with babel) inside of your Jest tests.
+
+Install it to your project:
+
+```shell
+npm install --save --dev jest-preset-hops babel-core@^7.0.0-0
```

Comment:
Installation isn't enough, right? We still need to set it as a preset for Jest.

Reply:
Indeed, nice catch!